### ISSUE TITLE
Could you please post a win32 version for testing on Windows?

### ISSUE THREAD

**Issue by xcom169** [Sude-/lgogdownloader#198]
Could you please post a win32 version for testing on Windows 10 terminal?

Thanks

**Comment by Sude-** [comment]
I don't have Windows installed currently so I can't build or test the downloader on Windows.

**Comment by ssokolow** [comment]
Assuming the license hasn't changed, https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/ still has Windows VMs that are licensed for "testing" without constraining it to testing websites and which grant you explicit permission to use a snapshot to bypass the expiry of the trial period.

As for building legally redistributable stuff, that's what a CI service like [AppVeyor](https://www.appveyor.com/) with Windows VMs free to open-source projects is for.

**Comment by shakeyourbunny** [comment]
About a year ago, I tried to build it, but it needs some porting effort, depending, what compiler you are using. I could not do it, because I'm not a programmer, more a self-taught jack of all trades

- With VisualC++ my roadblock was finding a proper JSON implementation.
- MSYS(2) there is some issue with getting the building enviroment right.
- Cygwin has some memory / malloc symbol stuff which I could not resolve.

Sude-, there is also a msys cross compiler which also works with Linux and you can do testing with the stuff ssokolow mentioned.

**Comment by ssokolow** [comment]
Good point. While you need real Windows to test properly, any Debian-family distro should have `mingw-w64` and `wine` for preliminary building and testing.
